plugin/cog/cog-rangereader-azure/pom.xml

```diff
@@ -0,0 +1,60 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>imageio-ext-cog-rangereader-azure</artifactId>
+
+    <parent>
+        <groupId>it.geosolutions.imageio-ext</groupId>
+        <artifactId>imageio-ext-cog</artifactId>
+        <version>1.4-SNAPSHOT</version>
+    </parent>
+
+    <name>Cloud Optimized GeoTIFF Azure Range Reader</name>
+
+    <profiles>
+        <profile>
+            <id>online</id>
+            <properties>
+                <online.skip.pattern>disabled</online.skip.pattern>
+            </properties>
+        </profile>
+    </profiles>
+
+    <dependencies>
+        <dependency>
+            <groupId>it.geosolutions.imageio-ext</groupId>
+            <artifactId>imageio-ext-cog-commons</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.microsoft.azure</groupId>
+            <artifactId>azure-storage-blob</artifactId>
+            <version>11.0.0</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
```

Comment:
A dependency on spring-web is a surprise

Reply:
... going to doublecheck

Reply:
It's being used by the client code to check the status of the HTTP Response through the HttpStatus class.

Reply:
It's being used by the client code to check the status of the HTTP Response through the HttpStatus class.

Reply:
Done